src/dropbox.js

```diff
@@ -381,31 +411,49 @@
       }
 
       //use _getFolder for folders
-      if (path.substr(-1) === '/') { return this._getFolder(path, options); }
+      if (path.substr(-1) === '/') {
+        return this._getFolder(path, options);
+      }
 
-      return this._request('GET', url, {}).then(function (resp) {
+      var params = {
+        headers: {
+          'Dropbox-API-Arg': JSON.stringify({path: getDropboxPath(path)})
+        }
+      };
+      if (options && options.ifNoneMatch) {
+        params.headers['If-None-Match'] = options.ifNoneMatch;
+      }
+
+      return this._request('GET', url, params).then(function (resp) {
         var status = resp.status;
         var meta, body, mime, rev;
-        if (status !== 200) {
+        if (status !== 200 && status !== 409) {
           return Promise.resolve({statusCode: status});
         }
-
+        meta = resp.getResponseHeader('Dropbox-API-Result');
         body = resp.responseText;
+
+        if (status == 409) {
```

Comment:
Is JSHint missing from the Travis runs now? I think that would complain on my machine.